internal/formats/spdx22json/test-fixtures/snapshot/TestSPDXJSONImagePresenter.golden

```diff
@@ -3,44 +3,15 @@
  "name": "user-image-input",
  "spdxVersion": "SPDX-2.2",
  "creationInfo": {
-  "created": "2021-10-22T19:25:38.341582Z",
+  "created": "2021-10-29T16:26:09.001799Z",
   "creators": [
    "Organization: Anchore, Inc",
    "Tool: syft-[not provided]"
   ],
   "licenseListVersion": "3.14"
  },
- "syftSourceData": {
```

Comment:
Moving to the new model it looks like we lose this `syftSourceData` information?

Is this something we no longer want to include in the output?

Reply:
I noticed this while preparing the PR --this field was never meant to be included to begin with. I added it accidentally in https://github.com/anchore/syft/pull/584 while removing features from https://github.com/anchore/syft/pull/578 . This is probably better served in its own PR (or at least in the PR description here!)